linuxdeploy-plugin-gtk.sh

```diff
@@ -67,6 +85,7 @@ search_tool() {
 }
 
 APPDIR=
+DEPLOY_GTK_VERSION="${DEPLOY_GTK_VERSION:-0}"
```

Comment:
Please add a comment that states that `0` indicates "not found".

Code style wise, I think you should move it above the `APPDIR` variable. The latter is set in the following while loop, and logically belongs to that block.

Reply:
Ok, fixed in cd303e99a97a61d599366dd71c98fdc334d4d4bc.